Gemfile.lock

```diff
@@ -41,6 +41,16 @@ GIT
       sprockets-rails (~> 2.0.0.backport1)
 
 GIT
+  remote: https://github.com/opf/openproject-translations.git
+  revision: f4b469f5ea2b9d0d9288a6d79f452198784c7abb
+  branch: dev
```

Comment:
Travis now complains rightfully, that this here requires `dev` while the Gemfile denotes to require `release/4.2`